src/runtime/security/init.cpp

```diff
@@ -68,5 +68,22 @@ bool init(bool is_server)
     return true;
 }
 
+bool init_for_zookeeper_client()
+{
+    error_s err = run_kinit();
+    if (!err.is_ok()) {
+        derror_f("initialize kerberos failed, with err = {}", err.description());
+        return false;
+    }
+    ddebug("initialize kerberos for zookeeper client succeed");
+
+    err = init_sasl(false);
```

Comment:
Better to make a simple refactor on the boolean parameter, boolean is meanless. For example add an enum like:
```
enum class KerberosRole {
    kClient = 0,
    kServer = 1
};
```